docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -29,6 +31,10 @@ An exemplar is a specific trace representative of measurement taken in a given t
 Since traces and metrics co-exist in the metrics-generator,
 exemplars can be automatically added, providing additional value to these metrics.
 
+[Exemplars](/docs/grafana-cloud/data-configuration/traces/exemplars/) are GA in Grafana Cloud so you can also push your own.
```

Comment:
Updating this link since the location has changed. 
```suggestion
[Exemplars](/docs/grafana-cloud/monitor-infrastructure/traces/exemplars/) are available in Grafana Cloud so you can also push your own.
```